.github/workflows/nix.yml

```diff
@@ -19,10 +19,10 @@ jobs:
         platform:
           - os: darwin
             cpu: x86_64
-            base: macos-13 # always x86_64-darwin
+            base: macos-15-intel # always x86_64-darwin
```

Comment:
Minor: Same note as below, I'd go for removing macOS on Intel vs keeping it.

Reply:
yes, let'go for it